### ISSUE TITLE
FR: enable LogoURL in the AccessApplication spec

### ISSUE THREAD

**Issue by psarossy** [BojanZelic/cloudflare-zero-trust-operator#106]
Pretty much what the title says, could the LogoURL be added from the API into the spec?

https://github.com/cloudflare/cloudflare-go/blob/55b6bd51bddffa69c78b40a2ede7a77174f6bcce/access_application.go#L32

**Comment by BojanZelic** [comment]
Makes sense, It should be a simple change, I'll try to add soon when I get a chance.

**Comment by BojanZelic** [comment]
completed by https://github.com/BojanZelic/cloudflare-zero-trust-operator/pull/111
